src/meta.ts

```diff
@@ -30,4 +30,6 @@ export const DEFAULT_LOCALE_COUNTRY_MAP = {
   ja: 'ja',
   es: 'es',
   vi: 'vn',
+  hu: 'hu',
```

Comment:
it looks like the Hungarian flag should be available by default:

https://github.com/lokalise/i18n-ally/blob/main/src/utils/flags.ts#L111
https://github.com/lokalise/i18n-ally/blob/main/res/flags/hu.svg
If it's not showing, then open a separate issue for it, please

```suggestion
```